### PR TITLE
Fix Windows CLI : remove '\\?\' windows path prefix in initArgs

### PR DESCRIPTION
**Pull request by elchead**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**
Fixes [#619](https://github.com/kyma-project/cli/issues/619)

image

### CODE REVIEW

File: provision/internal/operator/terraform/tfcmd.go [hunk 1]
```diff
@@ -50,7 +51,10 @@ func initArgs(p types.ProviderType, cfg map[string]interface{}, clusterDir strin
 		// TODO remove this condition when fully migrated to modules
 		if m := tfMod(p); m != "" {
 			args = append(args, fmt.Sprintf("-from-module=%s", m))
-		}
+		}}
```
Comment (clebs) [inline provision/internal/operator/terraform/tfcmd.go:54]: Please run `gofmt -s -w` no this file, otherwise the CI/CD validation will fail.

**Review by clebs — APPROVED** [review]

**Review by clebs — APPROVED** [review]

**Review by a-thaler — APPROVED** [review]

**Pull request merged**